v0/destinations/google_cloud_function/util.js

```diff
@@ -0,0 +1,63 @@
+const {
```

Comment:
Please format the file and install prettier extension

Reply:
fixed